### PR TITLE
added heartbeat on alertmanger level, inspired by engineering

### PR DESCRIPTION
**Pull request by ironoa**

### CODE REVIEW

File: config/charts/templates/kube-prometheus-stack.tpl [hunk 1]
```diff
@@ -60,9 +66,18 @@ alertmanager:
       webhook_configs:
       - url:  "http://matrixbot:8080/skill/alertmanager/webhook"
     {{#if opsgenieEnabled}}
+    - name: heartbeats
+      webhook_configs:
+      - http_config:
+          basic_auth:
+            password: {{ env "OPSGENIE_TOKEN" | default "opsgenie_token" }}
+        url: https://api.eu.opsgenie.com/v2/heartbeats/{{ deploymentName }}/ping
```
Comment (ironoa) [inline config/charts/templates/kube-prometheus-stack.tpl:74]: @pampatzoglou has this endpoint to be configured on opsgenie ?

File: config/charts/templates/kube-prometheus-stack.tpl [hunk 2]
```diff
@@ -60,9 +66,18 @@ alertmanager:
...
     - name: opsgenie
       opsgenie_configs:
-      - api_key: ''
```
Comment (ironoa) [inline config/charts/templates/kube-prometheus-stack.tpl:65]: @pampatzoglou Was the before situation: not configured ?

File: config/charts/templates/kube-prometheus-stack.tpl (hunk 1 above)
Comment (pampatzoglou) [reply config/charts/templates/kube-prometheus-stack.tpl:74]: no we need to manually goto the webUI of opsgenie and set them up https://web3foundation.app.eu.opsgenie.com/settings/heartbeat

File: config/charts/templates/kube-prometheus-stack.tpl [hunk 3]
```diff
@@ -95,3 +110,19 @@ kubeStateMetrics:
     requests:
       cpu: 10m
       memory: 16Mi
+
```
Comment (ironoa) [inline config/charts/templates/kube-prometheus-stack.tpl:113]: chart values: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L82

trying to replicate: https://github.com/w3f/base-services-charts/blob/master/charts/base-services/templates/rules/alertmanager.yaml#L33-L42

File: config/charts/templates/kube-prometheus-stack.tpl (hunk 2 above)
Comment (ironoa) [reply config/charts/templates/kube-prometheus-stack.tpl:65]: Correct, I'd say so

**Review by pampatzoglou — APPROVED** [review]

**Pull request merged**